example/src/JsonSchemaFormSuite.js

```diff
@@ -318,7 +318,7 @@ class JsonSchemaFormEditor extends React.Component<Props, State> {
                     <PredefinedGallery
                       schema={this.props.schema}
                       uischema={this.props.uischema}
-                      mods={{}}
+                      mods={this.props.mods}
```

Comment:
What was this change for?

Reply:
The mods weren't being passed into the PredefinedGallery in the example app -- I noticed that while testing my changes